song-server/src/test/java/bio/overture/song/server/utils/EndpointTester.java

```diff
@@ -160,9 +161,9 @@ public ResponseOption getAnalysisTypeVersionGetRequestAnd(
         .getAnd();
   }
 
-  // GET /schemas/meta
-  public ResponseOption getMetaSchemaGetRequestAnd() {
-    return initWebRequest().endpoint(SLASH.join(SCHEMAS, META)).getAnd();
+  // GET /schemas/registration
```

Comment:
don't need to change now, but proper java doc on public methods if you are going to comment